src/components/InternalDropdown/__test__/internalDropdown.spec.js

```diff
@@ -51,4 +148,36 @@ describe('InternalDropdown', () => {
             value: undefined,
         });
     });
+    it('should render svg and input elements when enableSearch is passed', () => {
+        const component = mount(
+            <InternalDropdown label="Picklist" enableSearch>
+                <PicklistOption label="Option 1" name="option1" />
+                <PicklistOption label="Option 2" name="option2" />
+                <PicklistOption label="Option 3" name="option3" />
+            </InternalDropdown>,
+        );
+        expect(component.find('svg').exists()).toBe(true);
+        expect(component.find('input').exists()).toBe(true);
+    });
+    it('should not render svg and input elements if enableSearch is not passed', () => {
+        const component = mount(
+            <InternalDropdown label="Picklist">
+                <PicklistOption label="Option 1" name="option1" />
+                <PicklistOption label="Option 2" name="option2" />
+                <PicklistOption label="Option 3" name="option3" />
+            </InternalDropdown>,
+        );
+        expect(component.find('svg').exists()).toBe(false);
```

Comment:
here we should be more precise too, since in the future we add another svg elment it will fail, then we shoul find the element itself ```find(SearchIcon)```